### DISCUSSION TITLE
100% தமிழ் Translation for gitextensions Available with me.

### DISCUSSION THREAD

**Discussion by TamilNeram** [gitextensions/gitextensions#11021] [Q&A]
### Feature description

I have translated gitextensions offline and completed and reviewed it. I tried adding Tamil/தமிழ் (Ta) language on “https://explore.transifex.com/git-extensions/git-extensions/”  but got rejected. 

Please let me know how to share the translated file with the team.  Attached those file for reference here. Also please remove txt extension.

![தமிழ்](https://github.com/gitextensions/gitextensions/assets/67970539/4cad526d-eea1-4b49-9318-a4e038962a18)
[தமிழ்.Plugins.xlf.txt](https://github.com/gitextensions/gitextensions/files/11628207/Plugins.xlf.txt)
[தமிழ்.xlf.txt](https://github.com/gitextensions/gitextensions/files/11628210/xlf.txt)

### Environment

- GitExtensions version: latest
- GIT version: 2.40
- OS version: windows 10/11
- .NET version: latest

**Comment by gerhardol** [comment]
The language is now approved in Transifex, you are added as a coordinator for the language.
You should be able to upload .xlf files.
Welcome

**Comment by RussKie** [comment]
Thank you @TamilNeram, I see the translations have been uploaded, and we'll ship it in the next release. 

Is this the image to represent the language in the language selection dialog?
![தமிழ்](https://github.com/gitextensions/gitextensions/assets/67970539/4cad526d-eea1-4b49-9318-a4e038962a18)

**Reply by RussKie** [reply]
You can also send a pull-request with your translation.

1. Update https://github.com/gitextensions/gitextensions/blob/master/.tools/transifex/.tx/config adding your new language
2. Pull the new/updated translations from Transifex by running https://github.com/gitextensions/gitextensions/blob/master/.tools/transifex/DownloadTransifexTranslations.ps1
3. Add the new language icon to https://github.com/gitextensions/gitextensions/tree/master/GitUI/Translation. The size has to be 225x150.

**Reply by gerhardol** [reply]
Talked to a Tamil friend about this: There is no unified Tamil flag. The Tamil Nadu state in India has an emblem, but that is not directly connected to the language. The Sri Lanka Tamil flags are highly controversial (and not including the Tamils in India), it would be like using the Taiwan flag for China. The flag above was used by a historic empire, also not including all.

The common 'language flag' is supposedly using an unique Tamil letter:
https://en.wiktionary.org/wiki/%E0%AE%B4

**Reply by TamilNeram** [reply]
Hi Everyone, thanks for your inputs. I have uploaded the translation in Transifex. There are a few more pending items to translate. I will finish them all and make the PR and also consider the Tamil special letter for flag.

**Reply by TamilNeram** [reply]
Hi all,
         I got some time during this week, I have finished updating and reviewing all the entries in the Transifex website. Also, I created two pull requests.   
Update config for Tamil #11040—Contains only the config file change. 
Tamil Translation update #11041-Contains the config file change and the translation files.  

I was not sure if the second one is required or not.  Feel free to approve one or both of PR and close this discussion. 

Thanks everyone for the guidance.
